src/cmd_line.rs

```diff
@@ -21,6 +21,14 @@ fn get_styles() -> Styles {
         .placeholder(styling::AnsiColor::Cyan.on_default())
 }
 
+fn is_a_tty() -> &'static str {
```

Comment:
I think this would be more desirable.

```rust
pub struct Tty {
    is_tty: bool,
}

impl Tty {
    fn status() -> Self {
        Self {
            is_tty: std::io::stdout().is_terminal(),
        }
    }

    fn as_str(&self) -> &'static str {
        if self.is_tty {
            "1"
        } else {
            "0"
        }
    }
}
```

Right now we are duplicating two functions for output reasons:

```rust
pub fn is_tty() -> bool {
    std::io::stdout().is_terminal()
}

fn is_a_tty() -> &'static str {
    use std::io::IsTerminal;

    if std::io::stdout().is_terminal() {
        "1"
    } else {
        "0"
    }
}
```

you could now replace from `is_tty()` to `Tty::status()`

and `is_a_tty()` to `Tty::status().as_str()`